lineman/app/components/thread_page/activity_card/discussion_title_edited.haml

```diff
@@ -0,0 +1,10 @@
+%article.thread-item{ng-controller: 'DiscussionTitleEditedItemController', aria-labelledby: 'discussion-{{discussion.id}}'}
```

Comment:
At some point we want to go in and refactor this so there's not so much duplication of markup... also I'd be amazed if we actually needed to use ng-controller here

A thing for another time, though.